migrations/20190515180630-updates-is-private.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface
+      .addColumn('Updates', 'isPrivate', {
+        type: Sequelize.BOOLEAN,
+        defaultValue: false,
```

Comment:
If we're not interested in `null` then let's add a `allowNull: false` on the column

Reply:
Added.